docs/db-migration-guide.ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Charmed Kubeflow Databases Migration Guide\n",
+    "\n",
+    "There are two databases in Charmed Kubeflow - `katib-db` and `kfp-db`. In Charmed Kubeflow 1.7 these MariaDB databases deployed by `charmed-osm-mariadb-k8s` charm. Since release 1.7 there is an option to relate charm to new MySQL database that is deployed by `mysql-k8s` charm. In cases where previous deployment has MariaDB a migration of database is required."
```

Comment:
I'd do something like this:

```suggestion
    "There are two databases in Charmed Kubeflow - `katib-db` and `kfp-db` which are backed by the MariaDB `charmed-osm-mariadb-k8s` charm. In Charmed Kubeflow 1.7 we introduced support for MySQL databases, which can be deployed using the `mysql-k8s` charm.
This document will guide you through the migration process from a MariaDB to a MySQL database."
```

Reply:
Let's not reference "This document". Here's my suggestion:

When upgrading to Charmed Kubeflow 1.7, Maria DB databases must be migrated to MySQL. There are two databases in Charmed Kubeflow - `katib-db` and `kfp-db`. Prior to 1.7, both were MariaDB databases backed by the `charmed-osm-mariadb-k8s` charm. In 1.7 they are backed by MySQL the `mysql-k8s` charm.

Reply:
I will modif.

Reply:
Updated.

Reply:
I think this still applies